src/RFC-0312_DANHighLevelSpecification.md

```diff
@@ -293,17 +292,46 @@ requires us to modify the
 `TransactionOutput` definition to include a `minimum_value_commitment` field, defaulting to zero, to capture this extra
 information.
 
-* The `ContractSpecification`UTXO MUST also include:
-    * The contract description,
-    * the asset issuer record
-    * the contract definition, as described below.
+* The `ContractConstitution`UTXO MUST also include:
+    * The contract name;
+    * the asset issuer record;
+    * the contract id;
+    * the contract definition;
+    * It MUST include a list of public keys of the proposed VNC;
+    * It MUST include an expiry timestamp before which all VNs must sign and agree to these terms (the [acceptance period]);
+    * It MAY include quorum conditions for acceptance of this proposal (default to 100% of VN signatures required);
+    * If the conditions will unequivocally pass, the acceptance period MAY be shortcut.
+    * There MAY be an initial reward that is paid to the VN committee when the UTXO is spent. This reward is simply included
+        in the value of the `ContractConstitution` UTXO.
+    * The UTXO MUST only be spendable by a multisig of the quorum of VNs performing [side-chain] initialisation. (e.g. a 3
```

Comment:
Additionally the emergency keys, and asset owner can spend checkpoints under specific conditions.